libs/storage/src/storage.ts

```diff
@@ -71,6 +71,12 @@ export class Storage<Store> extends EventBus<{
     return this.data;
   }
 
+  async reset(data: Store) {
+    this.data = data;
+    await this.commit(this.name, this.data);
+    this.emit('update', this.data);
+  }
```

Comment:
Will this commit changes to the store regardless of whether the data has actually changed?

Reply:
Yes. We're not checking if the data has changed or not in the commit func.

`
async commit(key: string, data: NoteStorage): Promise<void> {
      localStorage.setItem(key, JSON.stringify(data));
    },
`